app/views/purchases/_purchase.html.erb

```diff
@@ -0,0 +1,12 @@
+<tr>
+  <td ><%= purchase.name %></td>
+  <td>$<%= purchase.amount %></td>
+  <td > <%= purchase.category.name %></td>
+  <td >
+      <%= button_to "Delete", category_purchase_path(purchase.category_id, purchase.id), method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger" %>
+  </td>
+</tr>
```

Comment:
- [ ] Based on the requirements of this project, for a given category, the list of transactions(purchases) is presented ordered by the most recent. Kindly fix :wrench: this issue.

- [Optional] It'll equally be great if you can include the date of creation of each purchase and category as shown on the design. I  think can render your app more professional